build.gradle

```diff
@@ -91,7 +91,9 @@ subprojects {
 
     repositories {
         mavenLocal()
-
+        maven {
+            url = "https://download.corda.net/maven/corda-dependencies"
+        }
```

Comment:
Why do we need this new repository declared here?

I only see `com.r3.corda.ledger.utxo:contract-testin` added

which is not in `corda-dependencies`  artifactory location

Reply:
I picked these changes from Jenny's old branch so probably just left that in there by mistake.
Removing now.